go-ssb-bindings/vendor/go.cryptoscope.co/librarian/badger/index.go

```diff
@@ -19,6 +21,22 @@ import (
 	"go.cryptoscope.co/librarian"
 )
 
+// badger starts to complain >100k
+var batchFullLimit uint32 = 75000
+
+// NASTY TESTING HACK
```

Comment:
INDEED

Reply:
oh boy, i didn't even merge this yet: https://github.com/cryptoscope/librarian/pull/1